velox/exec/TaskDriverOperatorLifecycle.md

```diff
@@ -0,0 +1,96 @@
+# Task, Driver, Operator lifecycle
+
+__TL;DR__ Task outlives Drivers and Operators. There are circular references between
```

Comment:
In case there are some ownership semantics between these entities, it would be good to state what those are.

Reply:
By ownership I meant e.g. If Driver is responsible for creating and destroying Operators then Driver is the owner of Operators.

Reply:
Operators are created by the Task who then transfers the ownership to the Driver. In the end Driver owns the Operators, but they are created by the Task.